### DISCUSSION TITLE
맵이나 구조물들을 직접 3D툴로 모델링해서 넣었을때 질문!

### DISCUSSION THREAD

**Discussion by kimyang-sun** [naverz/zepeto-studio-kor#978] [5. Q&A]
안녕하세요! :)
맵, 구조물들을 3D 툴을 이용해서 직접 모델링한 후에 유니티에 불러와서 FBX 파일로 넣었는데
제페토 캐릭터를 소환하니까 구조물들이나 땅을 통과해버립니다.
각각의 오브젝트들마다 Mesh 콜라이더를 넣어주니까 오브젝트 모델링에 맞게 통과를 안하게 됐는데
구조물들을 불러올대마다 유니티에서 Mesh 콜라이더를 하나하나 계속 넣어주는 방법밖엔 없는걸까요??

**Comment by ssm00** [comment]
제가 정확히 이해했는지는 모르겠지만
처음에 가져오실때에는 Mesh콜라이더를 추가해주시고 이후 콜라이더가 추가된 모델을 하이라키에서 드래그 드롭하여 프리팹으로 만드셔서 사용하시면 이후에 그 모델을 사용하실때 콜라이더가 추가된채로 사용하실 수 있어요 도움이 되셨는지 모르겠네요 ㅎㅎ;

**Reply by kimyang-sun** [reply]
답변 감사합니다. :)
